src/controller/static_files.py

```diff
@@ -7,13 +8,12 @@
 
 @static_files.route('/', methods=['GET'], defaults={'path': 'index.html'}) # redirect initial homepage requests to static files controller
 
-# return any requested static files
 @static_files.route('/<path:path>', methods=['GET'])
+@login_required
```

Comment:
but why?

Reply:
I figured it would be easier if there is a guarantee that the user is logged in if they are looking at the UI. If the user isn't logged in, it would redirect them to the Google login page. Should help take out some frontend fluff.

Reply:
Discussed offline and agreed that public assets shouldn't be behind an authentication layer.